content/tasks.go

```diff
@@ -39,6 +41,42 @@ func NewTaskMarker(t TaskStatus) *TaskMarker {
 	}
 }
 
+// ParseTaskStatus parses a task status from a TO DO/DOING/DONE/etc. string.
+func ParseTaskStatus(status string) TaskStatus {
```

Comment:
I tried to keep this function in `internal/markdown/parsing.go` but I remember having some circular import error or something, so I moved it here.